src/main/java/org/opensearch/timeseries/model/Config.java

```diff
@@ -343,6 +370,21 @@ public void writeTo(StreamOutput output) throws IOException {
         output.writeInt(recencyEmphasis);
         output.writeInt(seasonIntervals);
         output.writeInt(historyIntervals);
+        if (customResultIndexMinSize != null) {
```

Comment:
Can you just to use writeOptionalInt and readOptionalInt that would writeBoolean true/false based on null check?

Reply:
sure, will update it